this patch fixes upstream bug http://bugs.mutt.org/3300 so that
mutt will be able to open pgp crypted/signed message with gpgme

--- a/init.c
+++ b/init.c
@@ -53,6 +53,10 @@
 #include <sys/wait.h>
 #include <sys/time.h>
 
+#if defined(CRYPT_BACKEND_GPGME)
+#include <gpgme.h>
+#endif
+
 #define CHECK_PAGER \
   if ((CurrentMenu == MENU_PAGER) && (idx >= 0) &&	\
 	    (MuttVars[idx].flags & R_RESORT)) \
@@ -3266,6 +3270,10 @@
 
   mutt_read_histfile ();
 
+#ifdef CRYPT_BACKEND_GPGME
+  gpgme_check_version (NULL);
+#endif
+
 #if 0
   set_option (OPTWEED); /* turn weeding on by default */
 #endif
